sourcegraph/sourcegraph
# sg: Add description field to sg commands

Add a `description` field to `sg` commands specified in `sg.config.yaml`

The description is displayed in the list of commands when running `sg run`

## Test plan


- Run `sg run`
- See that the command list now shows a description, when provided in `sg.config.yaml`
